packages/db/src/enum.ts

```diff
@@ -92,3 +92,22 @@ export const PrimaryCategory = {
 } as const;
 export type PrimaryCategory =
   (typeof PrimaryCategory)[keyof typeof PrimaryCategory];
+
+export const CountryCode = {
```

Comment:
As said before in our call, we can avoid the use of this enum and store at T0 the entire list of countries in the countries table.

We can create a seed script to do this job and take the list form some open data.
This can be useful in order to avoid adding step by step new countries into the enum file and store all the countries at db startup. 

The connection of each country to the other tables will be made using their mysql id. 
This can make everything easier, more lightweight and faster for the db when it runs each query.